### PR TITLE
Feature/update endpoint to fetch tweets by user

### PR DESCRIPTION
**Pull request by AndrewKaenko**

### CODE REVIEW

File: src/repositories/user.repository.ts [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
 import { User, UserModel } from "../models/User.model";
 
-export const getList = async (queryParams: object) => {
-  const query = {};
-  const options = {
-    ...queryParams,
-  };
-  const result = await UserModel.paginate(query, options);
+export const getList = async (query: object, options: object) => {
+  const result = await UserModel.paginate(query, { ...options });
```
Comment (VitaliiDenysiuk) [inline src/repositories/user.repository.ts]:
```suggestion
export const getList =  (query: object, options: object) => {
  return UserModel.paginate(query, { ...options });
```

File: src/models/Following.model.ts [hunk 2]
```diff
@@ -37,4 +37,4 @@ FollowingSchema.plugin(paginate);
 export const FollowingModel = mongoose.model<
   Following,
   mongoose.PaginateModel<Following>
->("Folowings", FollowingSchema, "Folowings");
+>("Folowing", FollowingSchema, "Folowing");
```
Comment (VitaliiDenysiuk) [inline src/models/Following.model.ts]:
```suggestion
>("Following", FollowingSchema, "Following");
```

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]

File: src/models/Following.model.ts (hunk 2 above)
Comment (VitaliiDenysiuk) [inline src/models/Following.model.ts]:
```suggestion
>("Following", FollowingSchema, "Following");
```
repeat second time

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]

**Review by VitaliiDenysiuk — APPROVED** [review]
looks good to me

**Review by fominmaksim — APPROVED** [review]

**Pull request merged**